spec/create-resource.md

```diff
@@ -0,0 +1,35 @@
+The *create resource* operation requests the creation of a new resource on the server.   The resource can be a non-container (data) resource or a container (collection) resource.  The operation can either let the server assign a new identifier within a target container (like creating a new file in a folder), or use a client-specified identifier (creating or overwrite at a specific path).
+
+**Inputs:**
+
+* **Target location:** Either a *target container identifier* (when the client wants the server to assign a name within that container) **or** a *full target resource identifier* (when the client specifies the exact desired identifier for the new resource).
+
+* **Resource content:** The content to store in the resource (binary or text). This is optional if creating an empty container or an empty resource.
+
+* **Suggested name:** An optional hint for naming (if the server will assign the identifier). For example, the client may propose a filename or resource name, which the server may use or adjust.
+
+* **Media type:** The MIME type or format of the content (e.g., `text/plain, application/json`). This helps the server understand how to store and serve the content. It may be mandatory for some protocols when content is provided (and may be omitted if creating an empty container with no content).
```

Comment:
```suggestion
* **Media type:** The MIME media type or format of the content (e.g., `text/plain`, `application/json`). This helps the server understand how to store and serve the content. It may be mandatory for some protocols when content is provided (and may be omitted if creating an empty container with no content).
```